src/diffpy/utils/parsers/serialization.py

```diff
@@ -40,13 +40,13 @@ def serialize_data(
 
     Parameters
     ----------
-    filename
-        Name of the file whose data is being serialized.
+    filepath
```

Comment:
```python
    for headerfile in tlm_list:
        # gather data using loadData
        hdata = loadData(headerfile, headers=True)
        data_table = loadData(headerfile)

        # check path extraction
        generated_data = serialize_data(headerfile...)
```

Here `headerfile` is used for `serialize_data(filepath, ...)`. `headerfile` prints `/Users/macbook/Downloads/dev/diffpy.utils/tests/testdata/dbload/e1.gr`